src/dotnetsheff.Api/GetAvailableFeedbackEvents/OneSpeakerTwoTalksParser.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace dotnetsheff.Api.GetAvailableFeedbackEvents
+{
+    public class OneSpeakerTwoTalksParser : ITalkParser
+    {
+        public IEnumerable<Talk> Parse(PastEvent pastEvent)
+        {
+            var oneSpeakerRegex = new Regex("This event will be (split|broken down) into (2|two) (parts|talks), (?<talk1>.+) and (?<talk2>.+) both being presented by (?<speaker>.+).");
```

Comment:
Why does `(?<speaker>.+).` have a `.` (zero or many) at the end of that group?

Reply:
regex matches the `.` when speaker name at end of sentence